src/legacy/ui/public/timefilter/index.ts

```diff
@@ -17,8 +17,16 @@
  * under the License.
  */
 
-export { TimeRange, RefreshInterval } from '../../../../plugins/data/public';
+import uiRoutes from 'ui/routes';
+import { registerTimefilterWithGlobalState, getTimefilterConfig } from './setup_router';
+import { Timefilter, TimeHistory } from '../../../core_plugins/data/public/timefilter';
 
-export { timefilter, Timefilter, registerTimefilterWithGlobalState } from './timefilter';
-export { timeHistory, TimeHistory } from './time_history';
-export { getTime } from './get_time';
+const config = getTimefilterConfig();
+
+export { Timefilter, TimeHistory, getTime } from '../../../core_plugins/data/public/timefilter';
+export const timeHistory = new TimeHistory();
+export const timefilter = new Timefilter(config, timeHistory);
```

Comment:
Are `timeHistory` and `timefilter` exported here ultimately going to be exported in the `setup` lifecycle contract in `core_plugins/data/public/timefilter`? 

Do you think it's worth doing that now and re-exporting those stateful services here instead? Or was that what you were referring to when you said `However, imports \ services are not rewired yet`?

Reply:
Yeah.

The PR that had timeHistory and timefilter instances moved and re-exported was too big IMO.
I wanted to split them in two, for reviewability :)